rpm/sofia-sip.spec

```diff
@@ -23,59 +26,59 @@ multimedia distribution, and multimedia conferences.
 
 
 %package glib
-Summary:    Glib bindings for Sofia-SIP
-Group:      Communications/Telephony and IM
+Summary:    Glib bindings for Sofia-SIP 
 Requires:   %{name} = %{version}-%{release}
-Requires(post): /sbin/ldconfig
-Requires(postun): /sbin/ldconfig
 
 %description glib
 GLib interface to Sofia SIP User Agent library.
 
 %package utils
 Summary:    Sofia-SIP Command Line Utilities
-Group:      Applications/Internet
 Requires:   %{name} = %{version}-%{release}
 
 %description utils
 Command line utilities for the Sofia SIP UA library.
 
 %package glib-devel
 Summary:    Glib bindings for Sofia SIP development files
-Group:      Development/Libraries
-Requires:   %{name} = %{version}-%{release}
-Requires:   sofia-sip-glib = %{version}-%{release}
-Requires:   sofia-sip-devel = %{version}-%{release}
+Requires:   %{name}-glib = %{version}-%{release}
+Requires:   %{name}-devel = %{version}-%{release}
+Requires:   pkgconfig
 
 %description glib-devel
 Development package for Sofia SIP UA Glib library. This package
 includes libraries and include files for developing glib programs
 using Sofia SIP.
 
-
 %package devel
 Summary:    Sofia-SIP Development Package
-Group:      Development/Libraries
 Requires:   %{name} = %{version}-%{release}
+Requires:   pkgconfig
 
 %description devel
 Development package for Sofia SIP UA library.
 
+%package doc
+Summary:   Documentation for %{name}
+Requires:  %{name} = %{version}-%{release}
+
+%description doc
+Documentation for %{name}.
+
+
 %prep
-%setup -q -n %{name}-%{version}/src
+%setup -q -n %{name}-%{version}/%{name}
 
 %build
-%configure --disable-static
+./autogen.sh
+%configure --disable-static --without-doxygen --disable-stun
 make %{?_smp_mflags}
```

Comment:
```suggestion
%make_build
```

Reply:
That adds -O which litters the output, didn't like it

Reply:
Arguable argument.
Using the same macro all over the other packages is a good practise.

Reply:
> Using the same macro all over the other packages is a good practise.

Arguable argument too))

Reply:
`%make_build` macro is used all over the other packages and it makes easy to change the behaviour at once. So it should be used.

Reply:
It makes sense to use it I agree.

Reply:
> `%make_build` macro is used all over the other packages and it makes easy to change the behaviour at once. So it should be used.

That didn't convince me. Blindly changing build options for a bunch of packages at once doesn't sound like a great idea to me. The consistency argument doesn't apply here because lots of specs don't have `%make_build` (and it could be played the other way around - let's get rid of all `%make_build` statements). And it looks like a non-standard macro specific to our build environment, meaning an obscure dependency on the local configuration. All in all, I choose not to use it.

Reply:
We have been replacing "make %{?_smp_mflags}" with %make_build in many spec files recently. %make_build has been available in upstream rpm macros since version 4.12 (https://github.com/rpm-software-management/rpm/commit/89df36524bace71decee4ab4f979d4ffb449c9a7).

Reply:
I wouldn't add -j without making sure first that makefile actually supports parallel builds))

Reply:
Ok guys, if all of you like `%make_build` so much then so be it.